### DISCUSSION TITLE
Поддержка техник тест дизайна

### DISCUSSION THREAD

**Discussion by alkoleft** [bia-technologies/yaxunit#421] [Планы развития]
* [Фаззинг](https://ru.wikipedia.org/wiki/%D0%A4%D0%B0%D0%B7%D0%B7%D0%B8%D0%BD%D0%B3)
* [Попарное тестирование](https://habr.com/ru/companies/otus/articles/592575/)
* Граничные значения
* Эквивалентное разбиение

**Comment by 1cgh** [comment]
для pairwise может пригодиться: https://github.com/microsoft/pict

**Reply by alkoleft** [reply]
Смотрел в его сторону, есть опыт использования?

**Reply by 1cgh** [reply]
Не, обычно онлайн-тулзу используем, а по этой часто в статьях встречал
